docpages/example_programs/interactions_and_components/context_menus.md

```diff
@@ -4,7 +4,7 @@ Context menus are application commands that appear on the context menu (right cl
 
 \image html context_menu_user_command.png
 
-\note This example sets the command as the type dpp::ctxm_user which can only be used by right clicking on a user. To make it appear on a message, you'll want to switch the type to dpp::ctxm_message and listen for the on_message_context_menu (dpp::message_context_menu_t) event.
+\note This example sets the command as the type `dpp::ctxm_user` which can only be used by right clicking on a user. To make it appear on a message, you'll want to switch the type to `dpp::ctxm_message` and listen for the `on_message_context_menu` (`dpp::message_context_menu_t`) event.
```

Comment:
I really do not agree with putting dpp references inside code-blocks in a note, They look like this image and it's REALLY contrasting as opposed to the non-code block versions. Again, I would like to hear other opinions on this.
![image](https://github.com/brainboxdotcc/DPP/assets/46899449/6ac7f700-c65a-40e6-bd1c-77b8c3301812)

I customised this so you can see the difference between non-code blocks with hyperlinks, code blocks and code blocks with hyperlinks.

Reply:
Hm yeah that's true, it doesn't do well with the note tag